maxim/apis/maxim_apis.py

```
@@ -1007,18 +1011,21 @@ def execute_prompt_chain_for_data(
         self,
         prompt_chain_version_id: str,
         input: str,
-        data_entry: Dict[str, Union[str, List[str], None]],
+        variables: Dict[str, Variable],
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)